modules/precisoBidAdapter.js

```diff
@@ -1,99 +1,94 @@
-import { logMessage, isFn, deepAccess, logInfo } from '../src/utils.js';
+import { isFn, deepAccess, logInfo, replaceAuctionPrice } from '../src/utils.js';
 import { registerBidder } from '../src/adapters/bidderFactory.js';
 import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
-import { config } from '../src/config.js';
+// import { config } from '../src/config.js';
 import { convertOrtbRequestToProprietaryNative } from '../src/native.js';
+import { getStorageManager } from '../src/storageManager.js';
+import { MODULE_TYPE_UID } from '../src/activities/modules.js';
 
 const BIDDER_CODE = 'preciso';
+const COOKIE_NAME = '_sharedid';
 const AD_URL = 'https://ssp-bidder.mndtrk.com/bid_request/openrtb';
+// const AD_URL = 'http://localhost:80/bid_request/openrtb';
 const URL_SYNC = 'https://ck.2trk.info/rtb/user/usersync.aspx?';
 const SUPPORTED_MEDIA_TYPES = [BANNER, NATIVE, VIDEO];
 const GVLID = 874;
 let userId = 'NA';
+let precisoId = 'NA';
+let sharedId = 'NA'
+
+export const storage2 = getStorageManager({ moduleType: MODULE_TYPE_UID, moduleName: BIDDER_CODE });
+export const storage = getStorageManager({ moduleType: MODULE_TYPE_UID, moduleName: 'sharedId' });
```

Comment:
Will have to ask about this in a JS meeting because never seen sharedId also accessed in a bid adapter.